### PR TITLE
Update dependencies and reduce warnings

### PR DESCRIPTION
**Pull request by amillert**

### CODE REVIEW

File: conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/bitcoin/BitcoinIndexer.scala [hunk 1]
```diff
@@ -3,25 +3,25 @@ package tech.cryptonomic.conseil.indexer.bitcoin
 import java.util.concurrent.Executors
 
 import scala.concurrent.{ExecutionContext, Future}
-import scala.concurrent.duration.FiniteDuration
 
 import cats.effect.{IO, Resource}
-import org.http4s.headers.Authorization
-import org.http4s.BasicCredentials
-import org.http4s.client.blaze.BlazeClientBuilder
+// import org.http4s.headers.Authorization
```
Comment (piotrkosecki) [inline conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/bitcoin/BitcoinIndexer.scala:8]: either uncomment or remove those lines - I suspect if it's commented it is not needed.

File: conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/bitcoin/BitcoinIndexer.scala [hunk 2]
```diff
@@ -120,8 +93,8 @@ class BitcoinIndexer(
       rpcClient <- RpcClient.resource(
         bitcoinConf.node.url,
         maxConcurrent = bitcoinConf.batching.indexerThreadsCount,
-        httpClient, // TODO: wrap it into retry and logger middleware
-        Authorization(BasicCredentials(bitcoinConf.node.username, bitcoinConf.node.password))
+        httpClient // TODO: wrap it into retry and logger middleware
+        // Authorization(BasicCredentials(bitcoinConf.node.username, bitcoinConf.node.password))
```
Comment (piotrkosecki) [inline conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/bitcoin/BitcoinIndexer.scala:97]: can you explain why is this commented?

File: project/Dependencies.scala [hunk 3]
```diff
@@ -102,6 +106,20 @@ object Dependencies {
     "org.julienrf" %% "endpoints-akka-http-server"    % Versions.endpoints
   )
 
+  private val tapirShared = Seq(
```
Comment (piotrkosecki) [inline project/Dependencies.scala:109]: can you remove tapir part from this PR? I don't want to mix things when it is not necessary

File: conseil-common/src/test/scala/tech/cryptonomic/conseil/common/bitcoin/BitcoinStubs.scala [hunk 4]
```diff
@@ -16,7 +16,7 @@ import tech.cryptonomic.conseil.common.rpc.RpcClient
   */
 trait BitcoinStubs {
 
-  implicit val contextShift: ContextShift[IO]
+  // implicit val contextShift: ContextShift[IO]
```
Comment (piotrkosecki) [inline conseil-common/src/test/scala/tech/cryptonomic/conseil/common/bitcoin/BitcoinStubs.scala:19]: why is this commented?

Comment (amillert) [reply conseil-common/src/test/scala/tech/cryptonomic/conseil/common/bitcoin/BitcoinStubs.scala:19]: `ContextShift` is not a thing in `cats effect` 3.0 anymore (https://typelevel.org/cats-effect/docs/migration-guide#contextshift)

Comment removed

File: project/Dependencies.scala (hunk 3 above)
Comment (amillert) [reply project/Dependencies.scala]: Yup, removed

File: conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/bitcoin/BitcoinIndexer.scala (hunk 1 above)
Comment (amillert) [reply conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/bitcoin/BitcoinIndexer.scala]: ok, uncommented and fixed

File: conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/bitcoin/BitcoinIndexer.scala (hunk 2 above)
Comment (amillert) [reply conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/bitcoin/BitcoinIndexer.scala]: types in the new version of API were incompatible; but I managed to fix it now

File: conseil-api/src/main/resources/application.conf [hunk 5]
```diff
@@ -19,7 +19,7 @@ conseil {
     keys: []
     keys+= ${?CONSEIL_API_KEY}
     
-    allow-blank: false
+    allow-blank: true
```
Comment (piotrkosecki) [inline conseil-api/src/main/resources/application.conf:22]: by default it should be `false`, please be careful when you commit you configuration 😛

**Review by piotrkosecki — APPROVED** [review]
left one last comment, but otherwise great job!

**Review by vishakh — APPROVED** [review]

**Pull request merged**